presto-docs/src/main/sphinx/release/release-344.md

```diff
@@ -0,0 +1,64 @@
+  threads used for processing dynamic filters on coordinator. ({issue}`5341`)
+* Extend environment variables substitution (`${ENV:variable-name}`) to work also when the substituted text
+  is part of the configuration value. ({issue}`4345`)  
+* Remove `fast-inequality-joins` configuration property. The fast inequality joins remain enabled. ({issue}`5375`)
+* Report dynamic filters collection duration. ({issue}`5434`)  
+* Improve query performance by propagating more detailed dynamic filters to coordinator. ({issue}`5340`)  
+* Improve performance and accuracy of {func}`approx_percentile`. ({issue}`5158`)
+* Improve performance of cross join queries. ({issue}`5276`)
+* Prevent potential query deadlock when query runs out of memory.  ({issue}`5289`)
+* Fix failure due to rounding error when casting between two `timestamp` types with precision higher than 6. ({issue}`5310`)
+* Fix failure due to rounding error when casting between two `timestamp with time zone` types with precision higher than 3. ({issue}`5371`)
+* Fix column pruning for `EXPLAIN ANALYZE` ({issue} `4760`)
+
+## Security Changes
+
+```{warning}
+The file-based system and catalog access controls have changed in ways that reduce or increase permissions. Please, read these release notes carefully.
+```
+
+* Change file-based catalog access control from deny to allow when table, schema, or session property rules are not defined. ({issue}`5039`)
+* Add missing table rule checks for table and view DDL in file-based system access control. ({issue}`5039`)
+* Add missing schema rule checks for create schema in file-based system access control. ({issue}`5039`)
+* Add session property rules to file-based system access control. ({issue}`5039`)
+* Add catalog regex to table and schema rules in file-based system access control. ({issue}`5039`)
+* Change create, rename, alter, and drop table in file-based system controls to only check for table ownership.  ({issue}`5039`)
+* Change file-based system access control to support to files without catalog rules defined.  In this case, all access to catalogs is allowed. ({issue}`5039`) 
+* Change file-based system and catalog access controls to only show catalogs, schemas, and tables a user has permissions on. ({issue}`5039`)
+* Change file-based catalog access control to deny all permissions inspection and manipulation checks. ({issue}`5039`)
+* Add [file-based group provider](/security/group-file). ({issue}`5028`)
+
+## Hive Connector Changes
+
+* Add `hive.security=allow-all` which allows to skip all authorization checks. ({issue}`5416`)
+* Improve query concurrency by listing data files more efficiently. ({issue}`5260`)
+* Fix Parquet encoding for timestamps before 1970-01-01. ({issue}`5364`)
+
+## Kafka Connector Changes
+
+* Expose message timestamp via ``_timestamp`` internal column. ({issue}`4805`)
+* Add predicate pushdown for ``_timestamp``, ``_partition_offset`` and ``_partition_id`` columns. ({issue}`4805`)
```

Comment:
Since the _timestamp is just being added (bullet above), should it be mentioned in this bullet too?

Reply:
Adding new internal command. And adding pushdown mechanism altogether are two separate things.
How would you phrase the bullet regarding push-down without mentioning `_timestamp` column?